pandas/io/formats/printing.py

```diff
@@ -252,7 +255,7 @@ def enable_data_resource_formatter(enable: bool) -> None:
         return
     from IPython import get_ipython
 
-    ip = get_ipython()
+    ip = get_ipython()  # type: ignore[no-untyped-call]
```

Comment:
Add mypy error above

Reply:
that can't be the right error message.  `standardize_mapping` doesn't appear in the call.  Please revise

Reply:
I resolved this in the latest commit.

Sorry about the force push. I need to rebase on `main` to fix a recent build error. I will just merge next time